include/sqlpp11/compat/cxx_std_ver.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#ifdef _MSVC_LANG
+#define CXX_STD_VER _MSVC_LANG
```

Comment:
Is the define `CXX_STD_VER` specific to sqlpp11?
If yes I would rather advocate to rename it to be more specific to prevent any name conflicts with anyone else that might also have decided on this name.
Something like `SQLPP_CXX_STD_VER`?


_PS: I know this just moves the macro, but the question holds_

Reply:
Prefixing the macro makes sense. I renamed the macro: CXX_STD_VER -> SQLPP_CXX_STD